cep-cache-output.md

```diff
@@ -0,0 +1,103 @@
+# CEP for the cache output in v1 recipes / rattler-build
+
+<table>
+<tr><td> Title </td><td> The cache output in v1 recipes / rattler-build </td>
+<tr><td> Status </td><td> In Discussion </td></tr>
+<tr><td> Author(s) </td><td> Wolf Vollprecht &ltw.vollprecht@gmail.com&gt; </td></tr>
+<tr><td> Created </td><td> Nov 27, 2024</td></tr>
+<tr><td> Updated </td><td> </td></tr>
+<tr><td> Discussion </td><td>  </td></tr>
+<tr><td> Implementation </td><td> rattler-build </td></tr>
+</table>
+
+## Abstract
+
+This CEP aims to define the cache output for v1 multi-output recipes.
+
+## Background
+
+Sometimes it is very useful to build some code once, and then split it into multiple build artifacts (such as shared library, header files, etc.). For this reason, `conda-build` has a special, implicit top-level build.
+
+There are many downsides to the behavior of `conda-build`: it's very implicit, hard to understand and hard to debug (for example, if an output is defined with the same name as the top-level recipe, this output will get the same requirements attached as the top-level).
```

Comment:
Not just that, tests under such an output will be silently skipped, which regularly trips people up: https://github.com/conda/conda-build/issues/4172